### ISSUE TITLE
PC-DOS 2000 not bootable after running setup

### ISSUE THREAD

**Issue by rderooy** [joncampbell123/dosbox-x#1502]
**Describe the bug**
As mentioned in https://github.com/joncampbell123/dosbox-x/issues/1468 there is a peculiarity to running the PC DOS 2000 setup.exe from CD-ROM in the emulated DOS that DOSBox-X provides. This issue does not occur when booting from a bootdisk (with cd-rom support) and running setup.exe.

If you let it do an install to a HDD image file, the disk image is not bootable afterwards. If you try to boot it, you get a stream of:
LOG:  718860062 ERROR CPU:Illegal Unhandled Interrupt Called 6

The workaround, before starting setup.exe, is to manually copy IBMBIO.COM and IBMDOS.COM from the CD-ROM to the root of the C: drive, and then it boots afterwards.

**To Reproduce**
Create a config with
```
[cpu]
APMBIOS=FALSE
```

Start DOSBox-X with the config and run:
```
IMGMAKE hdd.img -t hd_520
IMGMOUNT C hdd.img
IMGMOUNT D PC-DOS_2000.iso -t iso
D:
SETUP
```
After the install is finished, run
```
BOOT -L C
```
And the LOG will contain a continuous stream of 
 LOG:  718860062 ERROR CPU:Illegal Unhandled Interrupt Called 6

Now delete hdd.img and try this instead:
```
IMGMAKE hdd.img -t hd_520
IMGMOUNT C hdd.img
IMGMOUNT D PC-DOS_2000.iso -t iso
D:
COPY DOSFILES\IBMBIO.COM C:
COPY DOSFILES\IBMDOS.COM C:
SETUP
```
And after that, try to boot it, and it will work.

I looked at the LOG from the first run and the second run, and there don't appear to be any differences. All the same errors occur.

**Environment (please complete the following information):**
- Fedora 31
- DOSBox-X 0.83.1-git (today's build)

**Additional context**
Install LOG when running setup.exe WITHOUT first copying IBMBIO.COM and IBMDOS.COM.

```
LOG: IDE: index 0 slave=0
LOG: Using partition 0 on drive (type 0x06); skipping 63 sectors
LOG: FAT: BPB says 63 sectors/track 128 heads 512 bytes/sector
LOG: Mounted FAT volume is FAT16 with 65510 clusters
LOG: Mapping BIOS DISK C/H/S 520/128/63 as IDE 4160/16/63
LOG: IDE: index 0 slave=1
LOG: 43504164 ERROR BIOS:CMOS:WRite to unhandled register 17
LOG: 43504185 ERROR BIOS:CMOS:WRite to unhandled register 17
LOG: 43504221 ERROR BIOS:Disk 0 not active
LOG: 43987385 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=AD80
LOG: 43987557 ERROR MISC:DOS:Setting country code not supported
LOG: 43987667 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=B700
LOG: 43987738 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43987776 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43987814 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43987852 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43988688 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43988739 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43988774 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43989544 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43990007 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43991461 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43991496 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43992266 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43993338 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43995164 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43995199 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43995969 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43997043 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG: 43997992 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 44032455 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=4A12
LOG: 44032464 ERROR BIOS:INT1A:Undefined call B0
LOG: 44032671 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 44032878 ERROR DOSMISC:DOS:IOCTL Call 4 unhandled
LOG: 44045230 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 98657607 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 98659394 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 98665599 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 98683954 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 322333414 ERROR DOSMISC:DOS:IOCTL Call 5 unhandled
LOG: 322371177 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322372137 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322401570 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322402530 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322431035 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322431995 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322461411 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322462371 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322491024 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322491984 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322521701 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322522661 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322551926 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322552886 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322582339 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322583299 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322612590 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322613550 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322642839 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322643799 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322673263 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322674223 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322703556 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322704516 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322733786 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322734746 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322764312 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322765272 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322794694 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322795654 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322824723 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322825683 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322855029 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322855989 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322885089 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322886049 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322915077 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322916037 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322945081 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322946041 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322974804 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 322975764 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323004695 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323005655 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323035027 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323035987 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323065454 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323066414 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323095535 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323096495 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323125744 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323126704 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323156241 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323157201 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323186668 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323187628 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323217099 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323218059 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323247073 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323248033 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323277215 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323278175 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323307364 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323308324 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323337799 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323338759 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323368097 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323369057 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323398354 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323399314 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323428613 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 323429573 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG: 335638899 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=B700
LOG: APM BIOS call attempted. set apmbios=1 if you want power management
LOG: 336342178 ERROR BIOS:INT15:Unknown call ax=4100
```

Install LOG when first having copied IBMBIO.COM and IBMDOS.COM:
```
LOG: IDE: index 0 slave=1
LOG:   73581629 ERROR BIOS:CMOS:WRite to unhandled register 17
LOG:   73581650 ERROR BIOS:CMOS:WRite to unhandled register 17
LOG:   73581686 ERROR BIOS:Disk 0 not active
LOG:   74064840 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=AD80
LOG:   74065012 ERROR MISC:DOS:Setting country code not supported
LOG:   74065122 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=B700
LOG:   74065193 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74065231 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74065269 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74065307 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74066143 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74066194 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74066229 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74066999 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74067461 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74068915 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74068950 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74069720 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74070793 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74072619 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74072654 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74073424 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74074497 ERROR DOSMISC:DOS:IOCTL Call 11 unhandled
LOG:   74075446 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:   74110977 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=4A12
LOG:   74110986 ERROR BIOS:INT1A:Undefined call B0
LOG:   74111193 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:   74111400 ERROR DOSMISC:DOS:IOCTL Call  4 unhandled
LOG:   74123803 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  336613038 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  336614825 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  336621030 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  336640705 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  560295785 ERROR DOSMISC:DOS:IOCTL Call  5 unhandled
LOG:  560333563 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560334523 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560363942 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560364902 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560393448 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560394408 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560423790 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560424750 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560453440 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560454400 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560484108 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560485068 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560514348 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560515308 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560544759 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560545719 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560575007 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560575967 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560605245 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560606205 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560635674 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560636634 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560665938 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560666898 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560696221 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560697181 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560726774 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560727734 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560757159 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560758119 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560787216 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560788176 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560817522 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560818482 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560847607 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560848567 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560877698 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560878658 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560907730 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560908690 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560937453 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560938413 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560967288 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560968248 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560997646 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  560998606 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561028075 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561029035 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561058337 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561059297 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561088540 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561089500 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561118881 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561119841 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561149207 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561150167 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561179644 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561180604 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561209615 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561210575 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561239768 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561240728 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561269939 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561270899 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561300373 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561301333 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561330673 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561331633 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561360907 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561361867 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561391178 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  561392138 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=1203
LOG:  828358669 ERROR DOSMISC:DOS:INT 2F Unhandled call AX=B700
LOG: APM BIOS call attempted. set apmbios=1 if you want power management
LOG:  829061959 ERROR BIOS:INT15:Unknown call ax=4100
```

**Comment by Wengier** [comment]
I believe this is the last known compatibility issue with the PC DOS Setup. There is already a workaround for this, so I will look at it slightly later.

**Comment by rderooy** [comment]
It seems that the latest changes to the FAT support have made things worse for PC DOS 2000. Now it no longer boots after running through SETUP, even if you first copy IBMDOS.COM and IBMBIO.COM.

No filesystem corruption though, DOSBox-X is still able to mount the volume.

```
D:\DOSFILES>sys c:
Extended Error 1
```

**Comment by Wengier** [comment]
@rderooy The command "SYS C:" is NOT going to work because the usage is incorrect. The SYS command requires TWO parameters, not one ("Extended Error 1" basically means incorrect usage). So please use "SYS . C:" instead of "SYS C:".

As a result, with the latest code it is even simpler to make the PC DOS 2000 install work - just use "D:\DOSFILES\SYS D:\DOSFILES C:" instead of two COPY commands before running SETUP.

**Comment by rderooy** [comment]
@Wengier Thanks, I just tried it and indeed running SYS in that manner works around the problem. I thought it would pick up the files from the current directory if you did not specify one.

**Comment by rderooy** [comment]
I just tried it with the latest build, and the additional SYS command is no longer needed!

```
imgmake hdd.img -t hd_520
IMGMOUNT C hdd.img
IMGMOUNT D PC-DOS_2000.iso -t iso
D:
SETUP
```
After a reboot, you can now successfully boot with
```
IMGMOUNT C hdd.img
BOOT C:
```
